src/components/MainPage.tsx

```diff
@@ -1,14 +1,88 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { fetchFares } from '../helpers/ApiCallHelper';
 import TrainBoard from './TrainBoard';
 import UserInputFormArea from './UserInputFormArea';
 
+export type StationInfo = {
+    name: string;
+    latitude: number;
+    longitude: number;
+    aliases: string[];
+    id: number;
+    crs: string;
+    nlc: string;
+    isGrownStation : boolean; 
+    isSilverSeekStation : boolean;
```

Comment:
Do we need isGrown and isSilverseek? I don't think these will ever be used?

Reply:
deleted